tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -628,91 +628,69 @@ async def test_offer_endpoints(wallet_rpc_environment: WalletRpcTestEnvironment)
     env: WalletRpcTestEnvironment = wallet_rpc_environment
 
     wallet_node: WalletNode = env.wallet_1.node
-
-    client: WalletRpcClient = env.wallet_1.rpc_client
-    client_2: WalletRpcClient = env.wallet_2.rpc_client
-
+    wallet_1_rpc: WalletRpcClient = env.wallet_1.rpc_client
+    wallet_2_rpc: WalletRpcClient = env.wallet_2.rpc_client
     full_node_api: FullNodeSimulator = env.full_node.api
 
     await generate_funds(full_node_api, env.wallet_1, 1)
     await generate_funds(full_node_api, env.wallet_2, 1)
 
-    # Creates a CAT wallet with 100 mojos and a CAT with 20 mojos
-    await client.create_new_cat_and_wallet(uint64(100))
-    res = await client.create_new_cat_and_wallet(uint64(20))
-    assert res["success"]
-    cat_0_id = res["wallet_id"]
-    asset_id = bytes32.fromhex(res["asset_id"])
-    assert len(asset_id) > 0
-
-    # TODO: Investigate why farming only one block here makes it flaky
+    # Creates a CAT wallet with 20 mojos
+    res = await wallet_1_rpc.create_new_cat_and_wallet(uint64(20))
+    cat_wallet_id = res["wallet_id"]
+    cat_asset_id = bytes32.fromhex(res["asset_id"])
+    # TODO: Investigate why farming only two blocks here makes it flaky
```

Comment:
Because we are creating a tx (minting CAT) but not waiting for the tx to enter the mempool

Reply:
Will address this once we have a central place for our helpers like in #11535.